direct/src/dist/FreezeTool.py

```diff
@@ -1957,7 +1958,7 @@ def generateRuntimeFromStub(self, target, stub_file, use_console, fields={},
 
         if append_offset:
             # This is for legacy deploy-stub.
-            print("WARNING: Could not find blob header. Is deploy-stub outdated?")
+            warnings.warn("Could not find blob header. Is deploy-stub outdated?", DeprecationWarning)
```

Comment:
I don't think this is a deprecation, so this should not be flagged as a deprecation warning.

Reply:
I made the change to a UserWarning